etc/profile-m-z/vmware.profile

```diff
@@ -22,6 +22,7 @@ mkdir ${HOME}/.cache/vmware
 whitelist ${HOME}/.vmware
 whitelist ${HOME}/.cache/vmware
 include whitelist-common.inc
+whitelist-runuser-common.inc
```

Comment:
Missing include.

Is the socket in /run/user/$UID in a dir?

Reply:
which socket? vmware doesn't use any socket? it uses systemd services though. 

btw, rather than allowing specific syscalls? can specific system call sets be applied?

Reply:
>  rather than allowing specific syscalls? can specific system call sets be applied?

?? you mean groups?

Reply:
yea, the pre-defined sets of system calls.

Reply:
> which socket? vmware doesn't use any socket? it uses systemd services though.

what is the output of `ls /run/user/$UID -R`?

Reply:
[ls /run/user/$UID -R](https://gist.github.com/Neo00001/f9a79b8ff93afb950d1d2ceadbfd9c1d)

Reply:
>  I just had to run vmware without firejail once thereafter it worked again with firejail.

There is no obvious vmware socket, but it maybe need access to a systemd socket or something else.

Reply:
keep `whitelist-runuser-common.inc` for now. if this problem persists, then it can be omitted. btw,at that point I also enabled `seccomp.block-secondary`, could that be the problem?  after commenting out `seccomp.block-secondary`, vmware misbehaved until I launched it without firejail.

Reply:
If `seccomp` fails, `seccomp.block-secondary` will fail too.

Reply:
I thought `seccomp.block-secondary` will only disable all non-native ABIs for system calls.Now,I do realize my wrong inference.If `seccomp` is still enabled for native syscalls then for sure it's useless in this context.

Reply:
> it's useless

No, but if `seccomp !opl` fails w/o a log for a blocked syscall, vmware is one of the few prgorams which don't like it if they run under seccomp.

Reply:
by useless, I meant to say that it can't be used for vmware profile. I didn't say seccomp as a feature is useless.